rivers/yahoo-finance-prices/parser.js

```diff
@@ -0,0 +1,24 @@
+var _ = require('lodash'),
+    moment = require('moment-timezone');
+
+module.exports = function(body, options, temporalDataCallback, metaDataCallback) {
+    var config = options.config,
+        data = JSON.parse(body),
+        dtarray = data.query.created.split('T'),
+        dateString = dtarray.shift(),
+        timeString = dtarray.shift().split('Z').shift(),
+        date = moment(dateString + ' ' + timeString, 'YYYY-MM-DD HH:mm:ss'),
+        timestamp = date.unix(),
+        res = data.query.results.quote,
+        symbol = res.symbol,
+        fieldValues;
+
+    moment.tz.setDefault(config.timezone);
```

Comment:
Ok, the vars are grouped, but now there is a problem with this moment statement because it must occur before you create anything with moment (which you do above in the var declaration). 

The typical way to solve this issue in JS is to declare your var on a different statement than you initialize it. So:

```javascript
var date = undefined;
```

Then later:
```javascript
data = moment();
```